bindings/dotnet/openDAQ.Net/openDAQ.Net.CI.Test/OpenDAQ_CITests.cs

```diff
@@ -169,6 +169,68 @@ public void Test_0001_ListObjectDisposeTest()
         obj.PrintReferenceCount();
     }
 
+    [Test]
+    public void Test_0002_ListObjectWithParamsNoExceptionTest()
+    {
+        Console.WriteLine("> creating string-list with exception");
+
+        Assert.DoesNotThrow(() =>
+        {
+            //<TValue> cannot be inferred from the items
+            var list = CoreTypesFactory.CreateList<StringObject>("item 1", "item 2", "item 3", "item 4");
+
+            Assert.That(list, Is.Not.Null);
+            Assert.That(list.Count, Is.EqualTo(4));
+            Assert.That(list[0].CanCastTo<StringObject>(), Is.True);
+            Assert.That((string)list[0], Is.EqualTo("item 1"));
```

Comment:
I would check the other 3 items a well and drop the foreach with Console.WriteLine

Reply:
ok, can do. That loop is just for the log output.

Reply:
done